build.gradle

```diff
@@ -98,6 +98,7 @@ allprojects {
 		beanshellVersion = '2.0b5'
 		jaxbApiVersion = '2.3.1'
 		jaxbImplVersion = '2.3.0.1'
+		micrometerVersion = '1.1.1'
```

Comment:
1.1.3 is out.

Reply:
Updated to 1.1.3. I see 1.1.4 is planned for today. I will update to 1.1.4 before our release planned for next Friday.

Reply:
I saw micrometer v1.1.4 has been released today. I updated the PR to use 1.1.4.